fdbserver/fdbserver.vcxproj

```diff
@@ -68,6 +68,7 @@
     <ClCompile Include="LatencyBandConfig.cpp" />
     <ActorCompiler Include="OldTLogServer_4_6.actor.cpp" />
```

Comment:
4_6 actually has a reasonable story towards deprecation,  since it was never publicly released.  I'll start a conversation with Evan about if we could remove it for 7.0.   6_0 and 6_2 will likely be with us for quite some time...